common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -201,48 +195,61 @@ def test_fields_on_others_public_profile(self):
         Then I shouldn't see the profile visibility selector dropdown.
         Also `location`, `language` and `about me` fields are not editable.
         """
-        self.visit_other_profile_page(self.OTHER_USER, privacy=self.PRIVACY_PUBLIC)
-        self.visit_other_profile_page(self.MY_USER)
+        other_username, other_user_id = self._initialize_other_user(privacy=self.PRIVACY_PUBLIC)
+        self.log_in_as_unique_user()
+        profile_page = self.visit_profile_page(other_username)
+        profile_page.wait_for_public_fields()
+        self.assertFalse(profile_page.privacy_field_visible)
+        fields_to_check = self.PUBLIC_PROFILE_FIELDS
+        self.assertEqual(profile_page.visible_fields, fields_to_check)
+        self.assertEqual(profile_page.editable_fields, [])
+        self.verify_profile_page_view_event(other_user_id, visibility=self.PRIVACY_PUBLIC)
+
+    def _initialize_other_user(self, privacy=None):
+        """
+        Initialize the profile page for the other test user
+        """
+        username, user_id = self.log_in_as_unique_user()
 
-        self.other_profile_page.wait_for_public_fields()
-        self.assertFalse(self.other_profile_page.privacy_field_visible)
+        if privacy is None:
+            privacy = self.PRIVACY_PUBLIC
 
-        fields_to_check = self.PUBLIC_PROFILE_FIELDS
-        self.assertEqual(self.other_profile_page.visible_fields, fields_to_check)
+        # Set the user's birth year and privacy policy
+        self.set_birth_year(birth_year=1990)
```

Comment:
Must: it isn't necessary to set the birth year on all new users. Jim's changes to auto_auth mean that new users will be > 14 years old by default (and setting the birth year causes extra eventing).